Bugzilla/Elastic/Search.pm

```diff
@@ -90,8 +90,11 @@ sub data {
         die $@;
     }
     $self->_set_query_time($result->{took} / 1000);
+    my %stupid_name = (
```

Comment:
I'm not sure 'stupid_name' is necessarily the right variable name choice to try and merge into upstream someday. Why is it stupid? A code comment would be useful here.